docs/getting_started.rst

```diff
@@ -43,6 +71,21 @@ From here, we can also convert our ``Pauli`` object back to a dense numpy array
 Pauli Strings
 ------------------------
 
+Pauli strings, also known as Pauli words, are tensor-product combinations of Pauli matrices. For example, the following is a valid Pauli string:
+
+.. math::
+
+    \mathcal{\hat{P}} = \sigma_x \otimes \sigma_y \otimes \sigma_z
+
+where :math:`\otimes` denotes the tensor product, and we can more simply denote by
```

Comment:
sometimes called the [Kronecker product](https://en.wikipedia.org/wiki/Kronecker_product)